datapackage_pipelines_mojp/common/processors/sync.py

```diff
@@ -104,6 +113,14 @@ def _filter_row(self, row, resource_descriptor):
                                 raise Exception("language identifier not according to iso639 standard: {}".format(lang))
                     # delete the combined json lang field from the new_doc
                     del new_doc[lang_field]
+                # add lower case title to other languages
+                en_he = ["title_en", "title_he", "title_en_lc", "title_he_lc"]
+                for title in new_doc:
+                    if "title_" in title and title not in en_he:
+                        lang_title = new_doc[title]
+                        lang_title_lc = lang_title.lower()
+                        if lang_title_lc != lang_title:
+                            new_doc[title] = lang_title, lang_title_lc        
```

Comment:
logic should be much simpler:
```
new_doc["title_lc"] = {lang: title.lower() for lang, title in new_doc["title"].items()}
```
also, not sure why you are comparing with title_en / title_he